components/x-audio/readme.md

```diff
@@ -23,16 +35,22 @@ The [`x-engine`][engine] module is used to inject your chosen runtime into the c
 
 ## Usage
 
-The components provided by this module are all functions that expect a map of [properties](#properties). They can be used with vanilla JavaScript or JSX (If you are not familiar check out [WTF is JSX][jsx-wtf] first). For example if you were writing your application using React you could use the component like this:
+This component exports both the plain UI component and the default is the redux-connected component, ready to play audio.
 
 ```jsx
 import React from 'react';
+
+// Import the plain UI components
 import { Audio } from '@financial-times/x-audio';
 
-// A == B == C
-const a = Audio(props);
-const b = <Audio {...props} />;
-const c = React.createElement(Audio, props);
+// Render the UI component as usual
+<Audio {...props} />;
+
+
+// OR, Import the component bundled with the html audio player with redux
+import ConnectedPlayer from '@financial-times/x-audio';
```

Comment:
the default export is function, which you run on your side to obtain the functional audio player component. Maybe we call it `createAudioPlayer`?
```
const AudioPlayer = createAudioPlayer();
<AudioPlayer {...props} />;
```

It's a function so you don't end up initialising the store just by importing it

Reply:
Maybe` initAudio` is a better name? 🤔

Reply:
@robsquires that's what I thought but then got confused when I read this: https://github.com/Financial-Times/x-dash/blob/x-audio/components/x-audio/src/Audio.jsx#L6 🤔 

Hmm